src/cpp/cpp_typecheck_fargs.cpp

```diff
@@ -40,8 +38,8 @@ void cpp_typecheck_fargst::build(
   operands.clear();
   operands.reserve(function_call.op1().operands().size());
 
-  for(std::size_t i=0; i<function_call.op1().operands().size(); i++)
-    operands.push_back(function_call.op1().operands()[i]);
+  for(const auto &op : function_call.op1().operands())
+    operands.push_back(op);
```

Comment:
Rewrite this (and some lines above) as `operands = function_call.op1().operands();` ?